packages/widget/src/ui/WebComponent.tsx

```diff
@@ -32,17 +32,7 @@ const WidgetWithProvider = (props: WebComponentProps) => {
     return accumulator;
   }, {});
 
-  const { toasterProps, endpointOptions, apiURL, ...swapWidgetProps } =
-    realProps as WebComponentProps;
-  return (
-    <SwapWidgetProvider
-      toasterProps={toasterProps}
-      endpointOptions={endpointOptions}
-      apiURL={apiURL}
-    >
-      <SwapWidget {...swapWidgetProps} />
-    </SwapWidgetProvider>
-  );
```

Comment:
This is now a duplicate since `SwapWidget` already comes pre-wrapped with a `Provider`